sneak_peek.md

```diff
@@ -61,7 +66,8 @@ This is essentially a placeholder for the next release note ...
   + none
 
 * New/updated utility program
-  + none
+  + ncmpidump
+    It can now be used to dump BP formated file.
```

Comment:
Please add example usage and output of a BP files into README.ADIOS (or README.BP)